examples/federation_authority/dumps/example.json

```diff
@@ -193,6 +193,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_relying_party",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8000/oidc/rp/resolve"
```

Comment:
can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

Reply:
ok!